vtty.c

```diff
@@ -742,8 +744,11 @@ static void __exit vtty_exit(void)
 }
 
 module_param(user_break_timing, bool, 0444);
+MODULE_PARM_DESC(user_break_timing, " set true, if the vtty provider can do break signal timing, defaults to Y");
```

Comment:
Yeah, I'm always in for better documentation and explanation. I'll go and incorporate your suggestions into the header and the help text. 
Any suggestions for the naming instead of `user_break_timing` ? I always prefer more verbatim/descriptive identifiers, but I had no good idea...

Reply:
Maybe `support_timed_break?` But I leave this up to you, I'm not convinced my name is a better choice. If you add a bit of an explanation (and mention it next to TAG_BREAK_CTL) it should be enough.